Simple-Login-System-Exercise/src/Views/Login.php

```diff
@@ -0,0 +1,38 @@
+<?php use src\Controllers\LoginController; ?>
```

Comment:
I don' think this is the best practice for this use case. Maybe you can try sending the request direct to the controller instead?